step-function/get-files/src/main.py

```diff
@@ -27,34 +27,50 @@ def get_object_file_type(bucket, key):
     return None
 
 
-def lambda_handler(event, context):
-    bucket = environ['BUCKET']
-    response = S3_CLIENT.list_objects_v2(Bucket=bucket, Prefix=event['job_id'])
+def get_products(files):
+    return [{
+        'url': item['download_url'],
+        'size': item['size'],
+        'filename': item['filename']
+    } for item in files if item['file_type'] == 'product']
+
+
+def get_browse(files):
+    browse = [item for item in files if item['file_type'] in ('amp_browse', 'rgb_browse')]
```

Comment:
could do `if 'browse' in item['file_type']` to leave room for future types of browses

Reply:
the rtc gamma PR has hyphenated types, e.g. `amp-browse`
https://github.com/ASFHyP3/hyp3-rtc-gamma/pull/160/files#diff-8be91576c32694692cf90f1752b8c91fR108